index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Portfolio website</title>
+    <link rel="stylesheet" href="portfolio-main.css">
+  </head>
+  <body>
+    <nav class="nav-bar">
+      <div class="status-bar ">
+        <p>9:41</p>
+        <img class="cellular" src="images/cellular.png" alt="cellular Connection icon"/>
+        <img class="wifi" src="images/Wifi.png" alt="Wifi icon"/>
+        <img class="battery" src="images/battery.png" alt="Battery icon"/>
+      </div>
+      <div class="toolbar">
+        <p>Chere Lema</p>
+        <img src="images/normal_button.png" alt="Normal Button"/>
+      </div>
+    </nav>
+
+    <header class="headline">
+      <section >
+        <h1 class="primary-text">Hey There. I&rsquo;m Chere</h1>
+        <h2 class="support-text">I&rsquo;am a Software Developer</h2>
+        <p class="description">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+          don’t hestiate to contact me.</p>
+      </section>
+      <section class="social-icons">
+        <img src="images/github.png" alt="Github Icon"/>
+        <img src="images/Linkedin.png" alt="Linkedin Icon"/>
+        <img src="images/angel_list.png" alt="Angel list Icon"/>
+        <img src="images/twitter.png" alt="Twitter Icon"/>
+        <img src="images/medium.png" alt="Medium Icon"/> 
+      </section>
```

Comment:
- [ ] Excellent job adding the icons for your social media 💯 , The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using `href`. Kindly, use `<ul>` and `<li>` tags for each `<a>` tag. 😄